modules/apps/portal-search/portal-search-web/src/main/java/com/liferay/portal/search/web/internal/result/display/context/SearchResultSummaryDisplayContext.java

```diff
@@ -24,6 +24,10 @@ public String getAssetRendererURLDownload() {
 		return _assetRendererURLDownload;
 	}
 
+	public long getAssetRendererURLDownloadSize() {
```

Comment:
I would remove the `URL` part since this can be confusing if it's a URL or size. So this would be `getAssetRendererDownloadSize`.

I think ideally we'd rename these to `getAssetRendererDownloadURL` and `getAssetRendererDownloadSize` but I don't want to introduce any breaking changes with this ticket since it's not completely necessary.

Reply:
Makes sense! In that case, I updated just `getAssetRendererURLDownloadSize` -> `getAssetRendererDownloadSize` and left the original `getAssetRendererURLDownload` alone 👍